boards/nxp/frdm_imx93/frdm_imx93_mimx9352_a55.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "frdm_imx93-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "NXP i.MX93 A55";
+	compatible = "fsl,mimx93";
```

Comment:
maybe it would be a good idea to try and keep these consistent with Linux? i.e.: `compatible = "fsl,imx93-11x11-frdm", "fsl,imx93";`